libs/langchain/langchain/chains/structured_output/base.py

```diff
@@ -106,15 +113,17 @@ class RecordDog(BaseModel):
     return prompt | llm.bind(**llm_kwargs) | output_parser
 
 
-# TODO: implement mode='openai-tools'.
 def create_structured_output_runnable(
     output_schema: Union[Dict[str, Any], Type[BaseModel]],
     llm: Runnable,
     prompt: BasePromptTemplate,
     *,
     output_parser: Optional[Union[BaseOutputParser, BaseGenerationOutputParser]] = None,
-    mode: Literal["openai-functions", "openai-json"] = "openai-functions",
-    enforce_single_function_usage: bool = True,
+    enforce_function_usage: bool = True,
+    return_single: bool = True,
```

Comment:
Why is this option needed?

Reply:
openai tools returns a list otherwise

Reply:
Should we maybe hide this from the user as an option? It feels like a weird option in the API

Reply:
Can keep this as `return_single`